### ISSUE TITLE
Importing/Migrating data from GoAccess < 1.4

### ISSUE THREAD

**Issue by ms217** [allinurl/goaccess#1904]
As the title states, is it possible to import or migrate B+Tree data to the On-Disk Persistance Storage? I couldn't find any information about that. Is this possible at all?

**Comment by allinurl** [comment]
Yes, I still have plans on putting something together, not sure if I can push it out on the upcoming release as is almost out the door. If not in v1.4.1, more likely on v1.4.2. Thanks for the heads up!
